.github/workflows/release-java-sdk-to-maven-central.yml

```diff
@@ -53,10 +48,10 @@ jobs:
           java-version: '11'
           distribution: 'temurin'
           cache: 'gradle'
-      - name: Validate Gradle wrapper
-        uses: gradle/wrapper-validation-action@f9c9c575b8b21b6485636a91ffecd10e558c62f6
-      - name: Publish Pulumi Java SDK
-        uses: gradle/gradle-build-action@749f47bda3e44aa060e82d7b3ef7e40d953bd629
```

Comment:
gradle/gradle-build-action is deprecated in fabor of gradle/actions/setup-gradle